src/java/bobbybot/Storage.java

```diff
@@ -0,0 +1,70 @@
+package bobbybot;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
```

Comment:
Perhaps a header comment here for this class could help to explain the functionality of this class?